mesheryctl/internal/cli/root/system/context/create.go

```diff
@@ -45,7 +45,7 @@ var createContextCmd = &cobra.Command{
 }
 
 func init() {
-	createContextCmd.Flags().StringVarP(&url, "url", "u", "https://localhost:9081", "Meshery Server URL with Port")
+	createContextCmd.Flags().StringVarP(&url, "url", "u", "http://localhost:9081", "Meshery Server URL with Port")
```

Comment:
@navendu-pottekkat global variables / strict properties should be used here. Can you assist?

Reply:
Yes. @abhishekkumkar we could move the URL to the `helpers.go` file where a we also have other global variables.